cmd/repo_remove.go

```diff
@@ -25,8 +25,10 @@ func newRepoRemoveCmd(config *RootCommandConfig) *cobra.Command {
 	// initCmd represents the init command
 	var removeCmd = &cobra.Command{
 		Use:   "remove <name>",
-		Short: "Remove a configured Appsody repository",
-		Long:  ``,
+		Short: "Removes an Appsody repository.",
+		Long:  `Removes an Appsody repository from your list of configured Appsody repositories.`,
+		Example: `  appsody repo remove my-local-repo
+  Removes the "my-local-repo" repository from your list of configured repositories.`,
```

Comment:
Perhaps we should add this here..
The default repository cannot be removed from you list of configured Appsody repositories. To remove a default repository, you would first have to set a different default repository. (see appsody repo set-default)

Reply:
Or:
"You cannot remove the default repository, but you can make a different repository the default (see appsody repo set-default)."